### DISCUSSION TITLE
Bring back the working now.gg "proxy block" fix

### DISCUSSION THREAD

**Discussion by nvd123YT** [UseInterstellar/Interstellar#725] [Q&A] [answered]
Wanted to add to my fork and for general use
The file it was located in in the repository (frame.js) no longer exists

**Comment by 0mhx** [comment, marked as answer]
The fix is iffy and was very unreliable. There is a github issue you can follow: #552

**Comment by uhidontkno** [comment]
why are you trying to bring it back when it is unreliable and likely does not work?
